agent/src/beerocks/slave/backhaul_manager/internal/expected_ap_metrics_response.h

```diff
@@ -0,0 +1,97 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2016-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _EXPECTED_AP_METRICS_RESPONSE_H
+#define _EXPECTED_AP_METRICS_RESPONSE_H
+
+#include <bcl/beerocks_defines.h>
+#include <tlvf/CmduMessageTx.h>
+#include <tlvf/common/sMacAddr.h>
+#include <tlvf/tlvftypes.h>
+#include <unordered_set>
+
+namespace beerocks {
+
+class ExpectedApMetricsResponse {
+public:
+    /**
+     * @brief resets this object with a new mid
+     * @details resets all members of this object to its empty initial state except for the mid 
+     * that is set as was given bt the parameter (mid default value is zero (0))
+     * @param mid the new mid value
+     * @return radio mac of the found slave if found, otherwise empty string
+     */
+    void reset_to_new_mid(uint16_t mid = 0);
+
+    /**
+     * @brief returns the mid of this object
+     * @return mid
+     */
+    uint16_t get_mid() const;
+
+    /**
+     * @brief creates and returns a AP-RESPONSE-METRICS Tx message 
+     * @return a fresh new CmduMessage with the correct type and mid
+     */
+    ieee1905_1::CmduMessageTx &create_tx_message();
+
+    // bssid related operations
+
+    /**
+     * @brief stores expected bssids
+     * @details adds the bssids in the range [first,last) to the already existing internal list
```

Comment:
Unequal parenthesis: `[first,last)`
Edit: I see now that on a lot of places on documentation website do it as you did, not sure why. So fill free to ignore this comment.

Reply:
It's mathematical notation for a range that includes the beginning (square bracket) but doesn't include the end (round bracket).

Your comment, however, indicates that this is not knowledge that we can assume everyone has, so indeed it's better to replace it with
```suggestion
     *
     * Adds the bssids in the range @a first (inclusive) to @a last (exclusive) to the already existing internal list
```

Additional notes:
- capitalization, as @morantr already indicated;
- use `@a` to indicate an argument;
- don't use `@details` to separate from the brief, instead just leave an empty line.

Reply:
[ok, but let's make it explicit and not on comments basis](https://jira.prplfoundation.org/browse/PPM-44)